client/src/App.js

```diff
@@ -312,6 +330,18 @@ export default class NormalDisplay extends React.Component {
             running: true,
             logs: undefined
         });
+        //validate if journal ready
+        if(!this.state.journalReady){
+            const isJournalReady = await this.checkJournalReady();
+            this.setState({
+                journalReady: isJournalReady
+            });
+            if(!isJournalReady) {
+                const errMsg = 'Adobe I/O Event provider journal setup in progress. Please try again after 1 min';
```

Comment:
ah I see now, instead of polling you just display a message to try again after 1 min. I am ok with this instead of polling since we will change this behavior in the re-design ticket mentioned

Reply:
yes, for the redesign ticket, the consideration would be:
- waiting 60sec for setup, for all users
- initiate journal registration when `npm run start-dev` is called.